ci/jenkins/pipelines/prs/helpers/pr_manager/pr_manager.py

```diff
@@ -59,12 +59,7 @@ def filter_pr(args):
         repo = g.get_repo(GITHUB_REPO)
 
         pull = repo.get_pull(CHANGE_ID)
-        changed_files = pull.get_files()
-        files_list = []
-
-        # change paginated list to normal list
-        for f in changed_files:
-            files_list.append(f.filename)
+        files_list = [f.filename for f in pull.get_files()]
```

Comment:
I generally like this change, just wondering what it has to do with the objective of this PR?

Reply:
@Klaven I merged the PR before seeing your comment. Sorry :pray:

Reply:
@pablochacin it's ok. I did not block because I'm not as involved in this project as you yet. so until I am I just comment :)